packages/server/lib/browsers/cri-client.ts

```diff
@@ -226,10 +244,20 @@ export const create = async (target: websocketUrl, onAsynchronousError: Function
         })
       }
 
+      // Keep track of '*.enable' commands so they can be resent when
+      // reconnecting
+      if (command.endsWith('enable')) {
```

Comment:
nit: i am bothered more than i should be that this is `'enable'` and not `'.enable'`, even though it literally does not matter at all with the snakeCase commands

```suggestion
      if (command.endsWith('.enable')) {
```